pkg/devfile/adapters/docker/utils/utils.go

```diff
@@ -47,11 +71,12 @@ func DoesContainerNeedUpdating(component common.DevfileComponent, containerConfi
 	return !reflect.DeepEqual(devfileEnvVars, containerConfig.Env)
 }
 
-func AddProjectVolumeToComp(projectVolumeName string, hostConfig *container.HostConfig) *container.HostConfig {
+// AddVolumeToComp adds the project volume to the container host config
```

Comment:
Update the function description since it's not just the project volume that's getting added